app/src/main/java/cm/aptoide/pt/ApplicationComponent.java

```diff
@@ -26,4 +27,6 @@ ActivityComponent plus(ActivityModule activityModule,
   void inject(InstalledIntentService installedIntentService);
 
   void inject(PullingContentService pullingContentService);
+
+  void inject(DescriptionFragment descriptionFragment);
```

Comment:
This should be (and is being) injected in fragmentComponent

Reply:
Done